renovate.json

```diff
@@ -17,7 +17,13 @@
   "labels": ["dependencies"],
   "branchPrefix": "renovate-action/",
   "prConcurrentLimit": 2,
-  "packageRules": [],
-  "repositories": ["pagopa/interop-be-monorepo"],
-  "ignorePaths": ["docker/docker-compose.yml"]
+  "autodiscover": true,
+  "autodiscoverFilter": ["pagopa/interop-be-monorepo"],
+  "ignorePaths": ["docker/docker-compose.yml"],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["@protobuf-ts"],
+      "groupName": "protobuf-ts monorepo"
+    }
+  ]
```

Comment:
Even though the recommended config we extend includes `"group:monorepos"`, the `protobuf-ts` updates were not done together - see #355 and #356

Reply:
👍 

By the way `group:monorepos` works with an hardcoded list of popular monorepos (see https://docs.renovatebot.com/presets-monorepo/) so it is expected that we may specify additional ones.